core/services/assets/AssetManager.php

```diff
@@ -24,6 +23,8 @@
 abstract class AssetManager implements AssetManagerInterface
 {
 
+    const EE_NAMESPACE = 'eventespresso-';
```

Comment:
this needs to be a global const

Reply:
not a bad idea

Reply:
That can already be accessed globally as `AssetManager::EE_NAMESPACE`

Reply:
HA right... i had already been thinking about moving this into `/core/domain/Domain` which would:

- be more available for use since that class is loaded early on every request

- work better with add-ons since they all declare their own `Domain` class with details about themselves so makes a great standardized location for that constant

so when he said it needs to be a global const, i took that as affirming the above idea 🤦‍♂️

Reply:
updated to use `Domain::ASSET_NAMESPACE`

Reply:
awesome :taco: